types.go

```diff
@@ -203,3 +204,21 @@ func bearings(br []Bearing) string {
 	}
 	return strings.Join(s, ";")
 }
+
+type FallbackCoordinate string
+
+const (
+	FallbackCoordinateInput   FallbackCoordinate = "input"
+	FallbackCoordinateSnapped                    = "snapped"
```

Comment:
`FallbackCoordinateSnapped FallbackCoordinate` as well

Reply:
Yes, you are right. 
I used this like iota :)

Reply:
It doesn't work like iota. You must declare the type explicitly.
https://play.golang.org/p/iVeaitEtA2k